flags/14.0/vtctld.txt

```diff
@@ -336,26 +170,16 @@ max_rate_approach_threshold: 0.9
       --vtgate_grpc_crl string                                           the server crl to use to validate server certificates when connecting
       --vtgate_grpc_key string                                           the key to use to connect
       --vtgate_grpc_server_name string                                   the server name to use to validate server certificate
```

Comment:
I think we already deleted the `VtGateExecute` command. If yes, then we also don't need these `vtgate_` flags.

Reply:
fixed